custom/icds_reports/ucr/data_sources/migrations_form.json

```diff
@@ -143,21 +143,25 @@
       },
       {
         "column_id": "migration_status",
-        "datatype": "string",
+        "datatype": "small_integer",
```

Comment:
I have already asked Shubham to just create a new column.
Just out of curiosity, will the migration still break if we would have changed the column name? Because I think in that db will delete the existing column and create a new one? 
Also could that cause any problem like locking etc?

Reply:
@calellowitz